metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/MarcXmlEncoder.java

```diff
@@ -175,7 +175,8 @@ public void literal(final String name, final String value) {
         if (currentEntity.equals("")) {
             prettyPrintIndentation();
             writeRaw(String.format(CONTROLFIELD_OPEN_TEMPLATE, name));
-            writeEscaped(value.trim());
+            if (value!=null)
```

Comment:
I checked it, and can confirm it would have crashed before, if the `_else` is used. This is because of https://github.com/metafacture/metafacture-core/blob/252898009a5282a23de5e9d5da62482044178281/metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/MarcXmlHandler.java#L72 where the literal `type` is emitted with a `null` as value. Question is: should it be disallowed in general to emit a `null` value for literals? I could just add ` if (value!=null)` in Metamorph to disallow this for _all_ receivers (instead of only the `MarcXmlEncoder`). Re: test: ack.

Reply:
The `null` value issue has been discussed before; see #262 and #264.

Reply:
Thx for the pointer. So I will include a test for `MarcXmlHandler` as you proposed.